tracing-subscriber/tests/reload.rs

```diff
@@ -29,6 +29,17 @@ impl Subscriber for NopSubscriber {
     fn exit(&self, _: &Id) {}
 }
 
+pub struct NopLayer;
+impl<S: Subscriber> tracing_subscriber::Layer<S> for NopLayer {
+    fn register_callsite(&self, _m: &Metadata<'_>) -> Interest {
+        Interest::sometimes()
+    }
+
+    fn enabled(&self, _m: &Metadata<'_>, _: layer::Context<'_, S>) -> bool {
+        true
+    }
+}
```

Comment:
this should be moved into the `reload_filter` test if we add the `#[cfg(feature = "registry")]` attribute on that test; otherwise, this will be dead code if running tests with the "reload" feature but not the "registry" feature.
```suggestion
```